projects/storefrontlib/src/shared/components/table/table.component.html

```diff
@@ -1,40 +1,138 @@
 <ng-container *ngIf="tableData">
-  <table class="table cx-table">
-    <thead class="cx-thead-mobile">
-      <th scope="col" *ngFor="let column of columns">
-        {{ column.value }}
-      </th>
-    </thead>
-    <tbody>
-      <tr *ngFor="let row of tableData">
-        <td class="cx-code" *ngFor="let column of columns">
-          <div class="d-md-none cx-label">
-            {{ column.value }}
-          </div>
-          <label *ngIf="column.checkBox; else link">
-            <input
-              type="checkbox"
-              [checked]="row[column.key]"
-              (change)="onToggle($event, row, column.key)"
-            />
-          </label>
-          <ng-template #link>
-            <a
-              *ngIf="column.cxRoute && row; else text"
-              [routerLink]="
-                {
-                  cxRoute: column.cxRoute,
-                  params: row
-                } | cxUrl
+  <div class="desktop mode">
+    <table class="table cx-table">
+      <thead>
+        <th scope="col" *ngFor="let column of columns">
+          {{ column.value }}
+        </th>
+      </thead>
+      <tbody>
+        <tr *ngFor="let row of tableData">
+          <td class="cx-code" *ngFor="let column of columns">
+            <ng-container
+              *ngTemplateOutlet="
+                content;
+                context: { data: { row: row, column: column } }
               "
-              class="cx-value"
-            >
-              {{ row[column.key] }}
-            </a>
-          </ng-template>
-          <ng-template #text>{{ row[column.key] }}</ng-template>
-        </td>
-      </tr>
-    </tbody>
-  </table>
+            ></ng-container>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+  <div class="mobile mode">
+    <div class="cx-row" *ngFor="let row of tableData">
+      <div
+        *ngFor="let column of columns"
+        [ngClass]="{ 'cx-action': column?.checkBox || column?.button }"
```

Comment:
don't see results and even benefits from that

Reply:
Left from previous version, we don't need that for now.